app/views/exercise_solutions/_contextualization_results_body.html.erb

```diff
@@ -1,5 +1,5 @@
-<% if contextualization.results_visible? %>
-  <div>
+<div>
```

Comment:
I changed the name of this partial as the title is always shown and I found it confusing